lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-toggle {
+  display: flex;
+  margin-bottom: $nx-spacing-md;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &.nx-toggle--disabled {
+    &, &:hover {
+      color: $nx-text-color-disabled;
+      cursor: default;
+
+      .nx-toggle__control {
+        background-color: $nx-grey-200;
+        border-color: $nx-grey-500;
+      }
+      .nx-toggle__indicator {
+        background-color: $nx-grey-500;
+      }
+    }
+  }
+
+  &:focus-within {
+    .nx-toggle__control {
+      border-color: $nx-focus-border-color;
+      filter: drop-shadow(0 0 3px rgba($nx-focus-border-color, 0.4));
+
+      &:hover {
+        border-color: $nx-grey-1100;
+      }
+    }
+  }
+}
+
+.nx-toggle__input {
+  margin: 0;
+  position: absolute;
+  width: 0;
+  z-index: -1;
+
+  &:checked {
+    + .nx-toggle__control {
+      background-color: $nx-blue-600;
+
+      .nx-toggle__indicator {
+        background-color: #fff;
+        margin-left: 20px;
+      }
+    }
+  }
+}
+
+.nx-toggle__content {
+  flex-grow: 1;
+}
+
+.nx-toggle__control {
+  background-color: #fff;
+  border: 1px solid $nx-blue-600;
+  border-radius: 12px;
+  box-sizing: border-box;
+  display: flex;
+  height: 24px;
+  justify-content: start;
```

Comment:
Not necessary

Reply:
> Not necessary

https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/4ed34a543e2d7ed7e33e5362e4364120d7232936